woocommerce/woocommerce
# Update style of product attributes tab empty state

### All Submissions:






### Changes proposed in this Pull Request:



This PR updates the style of the product attributes tab when there are no attributes assigned to the product.

<img width="796" alt="Screenshot 2023-03-24 at 13 00 42" src="https://user-images.githubusercontent.com/2098816/227592025-26e1e2ad-09ff-4ba3-8b5d-6a268ae16f72.png">

Specifically:

* the buttons are moved to the top left
* the icon is updated


Closes #37151.



### How to test the changes in this Pull Request:




1. Add new product
2. Go to attributes tab
3. Verify style of empty state



### Other information:




### FOR PR REVIEWER ONLY:
